packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -1135,14 +1140,35 @@ class DoNothingIntent extends Intent {
 /// Attaching a [DoNothingAction] to an [Actions] mapping is one way to disable
 /// an action defined by a widget higher in the widget hierarchy.
 ///
+/// If [shouldHandleKey] is false, then not only will this action do nothing,
+/// but it will also terminate the propagation of the key used it trigger it (if
```

Comment:
Wondering whether we should clarify what propagation is terminated? In a sense, all actions terminate propagation to surrounding actions, no?